third_party/move/move-vm/metrics/src/lib.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) The Move Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use once_cell::sync::Lazy;
+use prometheus::{register_histogram_vec, HistogramTimer, HistogramVec};
+
+pub trait Timer {
+    fn timer_with_label(&self, label: &str) -> HistogramTimer;
```

Comment:
Given this is a `pub trait`, could we add a brief doc string for this method? Also, some guidance regarding the naming of the label, and any notes on performance considerations.

Reply:
Good point, done